recipes/DataProfiler/meta.yaml

```diff
@@ -0,0 +1,77 @@
+{% set name = "dataprofiler" %}
+{% set version = "0.8.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/DataProfiler-{{ version }}.tar.gz
+  sha256: ebd710d9bdd61cd2a66a5719ed3cc6a0442676191de299cdc978a5ec54ada813
+
+build:
+#  noarch: python
```

Comment:
Why not?

Reply:
@ocefpaf - Let me give it another shot - it gave me some errors when I tried noarch last time.

Reply:
Ok Now I remember, with no arch it started failing with different errors - on Linux ,vs Windows vs OSX. 
Linux Only Worked fine..